lib/minitest/reporters/html_reporter.rb

```diff
@@ -75,8 +75,8 @@ def initialize(args = {})
         @reports_path = File.absolute_path(reports_dir)
 
         puts "Emptying #{@reports_path}"
-        FileUtils.remove_dir(@reports_path) if File.exists?(@reports_path)
         FileUtils.mkdir_p(@reports_path)
+        File.delete(*Dir.glob("#{@reports_path}/#{@output_filename}"))
```

Comment:
Good point. I used the same code as used in the JUnit reporter (which has the possibility of generating more files). I'll amend that.